spec/controllers/client_api/users/user_teams_controller.rb

```diff
@@ -10,20 +10,20 @@
       FactoryGirl.create :user_team, team: @team, user: @user_one, role: 2
     end
 
-    it 'Returns HTTP success if user can leave the team' do
+    it 'should return HTTP success if user can leave the team' do
```

Comment:
Should is an empty word so you can delete should. it "should ..." was used in rspec 2. With rspec 3+ it is suggested that you write examples/tests like this => it "returns ..."